pkg/virt-controller/watch/migration.go

```diff
@@ -458,6 +458,14 @@ func (c *MigrationController) updateStatus(migration *virtv1.VirtualMachineInsta
 				}
 			}
 		case virtv1.MigrationScheduling:
+			if isPodPendingUnschedulable(pod) {
+				c.recorder.Eventf(
+					migration,
+					k8sv1.EventTypeWarning,
+					MigrationTargetPodUnschedulable,
+					"migration target pod for VMI [%s/%s] is at unschedulable phase.", vmi.Namespace, vmi.Name)
```

Comment:
I think we should avoid the word "phase" here and simply say that the migration target pod is currently unschedulable

Reply:
Done